unused.js

```diff
@@ -36,19 +36,72 @@ const entry = [
     '**/owa-immersive-bizchat-bootstrap/**',
 ];
 
+const entries = [
```

Comment:
We shouldn't be hard coding these values. They should be passed in

Reply:
This file is only a utility to test the tool against client-web, you can think of this file as a representation of how to use this tool inside owa-build. Whenever we integrate this inside owa-build this values will not be hardcoded.

Reply:
agree with jared here

Reply:
yeah this scratch file arguably doesn't belong to this project, but it checked in here for convenience. I think it's fine to leave in place for now, but you should probably have a comment at the head on the file describing the intent